oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/web/authentication/OAuth2EndpointUtils.java

```diff
@@ -94,5 +94,7 @@ static String normalizeUserCode(String userCode) {
 		sb.insert(4, '-');
 		return sb.toString();
 	}
-
+	static boolean validateUserCode(String userCode){
+		return userCode != null && userCode.replaceAll("[^A-Z\\d]+", "").length() == 8;
+	}
```

Comment:
Please format this method according to code style conventions.

Reply:
Hi @sjohnr ,
I have applied the .editorconfig to the file. Is there any other formatting that I need to do?

Reply:
I don't see anything after scanning it briefly. I'll get this merged this week. Thanks!